static/app/views/dashboardsV2/datasetConfig/base.tsx

```diff
@@ -47,6 +49,17 @@ export interface DatasetConfig<SeriesResponse, TableResponse> {
     meta: MetaType,
     contextualProps?: ContextualProps
   ) => ReturnType<typeof getFieldRenderer> | null;
+  /**
+   * Generate the request promises for fetching
+   * tabular data.
+   */
+  getTableRequest?: (
+    widget: Widget,
```

Comment:
I think it's a great opportunity for some cleanup if we can get away with not passing `widget` as an arg here! For example, in errors and transaction:
- is widget.name really necessary when building the event view (i'm not sure, but we should check)
- for the display type: 1) for the url, can we split world map requests and table requests and 2) have the "orchestrater" pass referrer as an optional arg so other requests can also set the referrer on their request if needed?

In the releases widget, we rely on widget.displayType (i think can be eliminated because in the context of getTableRequest, we know its table or big number) and widget.interval, i don't think we actually set it anywhere so can we just like hardcode that to '5m'?